commit/report.go

```diff
@@ -18,16 +20,15 @@ func (p *Plugin) Reports(seqNr uint64, outcomeBytes ocr3types.Outcome) ([]ocr3ty
 		return nil, fmt.Errorf("failed to decode Outcome (%s): %w", hex.EncodeToString(outcomeBytes), err)
 	}
 
-	// Reports are only generated from "ReportGenerated" outcomes
-	if outcome.OutcomeType != ReportGenerated {
-		return []ocr3types.ReportWithInfo[[]byte]{}, nil
-	}
-
-	rep := cciptypes.NewCommitPluginReport(outcome.RootsToReport, outcome.TokenPrices, outcome.GasPrices)
+	rep := cciptypes.NewCommitPluginReport(
+		outcome.MerkleOutcome.RootsToReport,
+		outcome.TokensOutcome.TokenPrices,
+		outcome.FeesOutcome.GasPrices,
+	)
```

Comment:
I think this will create a potentially empty report no?

In order to stay consistent at least with the previous code and before we add new logic, lets check

```golang
// TODO: add logic for gas/token price reporting, they should be reported even
// if merkle roots are empty if there is something to report.
if outcome.MerkleOutcome.OutcomeType != merkleroots.ReportGenerated {
  return nil, nil // nothing to report yet
}
```